Source/Scene/Model.js

```diff
@@ -1434,12 +1471,24 @@ define([
             parsePrograms(model);
             parseTextures(model);
         }
-
         parseMaterials(model);
         parseMeshes(model);
         parseNodes(model);
     }
 
+    function usesExtension(model, extension) {
+        var cachedExtensionsUsed = model._extensionsUsed;
+        if (!defined(cachedExtensionsUsed)) {
+            var extensionsUsed = model.gltf.extensionsUsed;
+            cachedExtensionsUsed = {};
+            var extensionsLength = extensionsUsed.length;
+            for (var i = 0; i < extensionsLength; i++) {
+                cachedExtensionsUsed[extensionsUsed[i]] = i;
```

Comment:
Just `= true` would be cleaner since the index doesn't matter once the data structure is created.